test/test.js

```diff
@@ -248,7 +248,57 @@ describe('Validator, Chain, and SDK work end to end', function () {
         const { hash } = await tableland.setController('0x14dC79964da2C08b23698B3D3cc7Ca32193d9955', name);
 
         expect(typeof hash).toEqual('string');
-        expect(hash.length).toEqual(56);
+        expect(hash.length).toEqual(66);
+    });
+
+    test('get controller returns an address', async function () {
+        const wallet = new Wallet('0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d' /* Hardhat #1 */);
+        const provider = new providers.JsonRpcProvider('http://localhost:8545');
+        const signer = wallet.connect(provider);
+
+        const tableland = await getTableland(signer);
+
+        const prefix = 'test_create_getcontroller';
+        // `key` is a reserved word in sqlite
+        const { name } = await tableland.create('keyy TEXT, val TEXT', { prefix });
+
+        const chainId = 31337;
+        // Hardhat #7
+        const controllerAddress = '0x14dC79964da2C08b23698B3D3cc7Ca32193d9955'
```

Comment:
Why hardcoded versus just pulling from the accounts list?

Reply:
This repo doesn't have hardhat as a dependency.  Should we add it so we can get the accounts?